pkg/report/linux_test.go

```diff
@@ -133,6 +134,11 @@ func TestLinuxSymbolizeLine(t *testing.T) {
 			"[   50.419727] 	baz+0x101/0x200\n",
 			"[   50.419727] 	baz+0x101/0x200 baz.c:100\n",
 		},
+		// Another KASAN frame format.
+		{
+			"[   50.419727][ T3822] baz+0x101/0x200\n",
```

Comment:
This test case is unrelated to the new functionality you are adding, correct?
Please consider adding it in a separate patch.